Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -65,9 +65,9 @@ public static void HandleDebugTeleportXYZ(Session session, params string[] param
         [CommandHandler("grantxp", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld, 1)]
         public static void HandleGrantXp(Session session, params string[] parameters)
         {
-            uint xp = 0;
+            ulong xp = 0;

```

Comment:
-Changed the XP variable to an unsigned long as we do not use negative values for XP.